sdk/react/panels/videoView.js

```diff
@@ -155,11 +156,12 @@ var VideoView = React.createClass({
       }} />);
   },
 
-  _renderPlaceholder: function() {
+  _renderPlaceholder: function(adOverlay) {
```

Comment:
Does it necessarily have to be part of the placeholder? can you just add this below the overlay?